### PR TITLE
Import Unicode Converter

### PR DESCRIPTION
**Pull request by Sarabveer**
- [x] Import mappings
- [x] Import converter
- [x] Add tests

### CODE REVIEW

File: lib/mappings.json [hunk 1]
```diff
@@ -1,85 +1,100 @@
...
+  "|":"ਙ",
+  "Î":"੍ਯ",
+  "ç":"੍ਚ",
+  "¨":"ੂ",
+  "Ø":"",
+  "ˆ":"ਂ",
+  "¤":"ੱ",
+  "œ":"੍ਤ",
+  "Ç":"☬"
+}
```
Comment (Harjot1Singh) [inline lib/mappings.json]: Please add trailing newline 📃

Comment (Sarabveer) [reply lib/mappings.json]: Done

**Comment by Sarabveer** [comment]
[Re-write GurbaniNow Gurmukhi Utils to JS](https://trello.com/c/B2Dvtic8/62-re-write-gurbaninow-gurmukhi-utils-to-js)

**Comment by Sarabveer** [comment]
LGTM 👍

**Pull request merged**